oidc-authenticator/src/main/java/org/xwiki/contrib/oidc/auth/internal/OIDCClientConfiguration.java

```diff
@@ -242,6 +242,11 @@ public Map<String, Set<String>> getProviderMapping()
      * @since 1.16
      */
     public static final String PROP_SCOPE = "oidc.scope";
+    
+    /**
+     * @since 2.5.2
```

Comment:
Since it's a new feature, I would make it 2.6.0.

Reply:
Ok, i changed it.